src/PointwiseFunctions/GeneralRelativity/WeylMagnetic.hpp

```diff
@@ -0,0 +1,76 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace gr {
+
+/// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes the magnetic part of the Weyl tensor.
+ *
+ * \details Computes the magnetic part of the Weyl tensor \f$B_{ij}\f$
+ * as:
+ *
+ * \f{align}{
+ * B_{ij} =
+ * \left(1/\sqrt{\det\gamma}\right)D_{k}K_{l(i}\gamma_{j)m}\epsilon^{mlk} \f}
```

Comment:
[optional while removing noexcepts] you could do `\frac{1}{\sqrt{\det\gamma}}` if you think that looks better

Reply:
Hi Nils, I didn't see this message. Would u want me to make this change?